### PR TITLE
WIP: Start nuking old framework support

### PR DESCRIPTION
**Pull request by puiterwijk**
Signed-off-by: Patrick Uiterwijk <patrick@puiterwijk.org>

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -3,8 +3,16 @@ NEWS
 ====
 
 :Authors: Toshio Kuratomi, Luke Macken, Ricky Elrod, Patrick Uiterwijk, Ralph Bean
-:Date: 5 May 2017
-:Version: 0.10.x
+:Date: TBD
+:Version: 0.11.x
```
Comment (pypingou) [inline NEWS:7]: I'd argue for a 1.0.x release here since it's quite non-backward compatible as change :D

File: NEWS [hunk 2]
```diff
@@ -3,8 +3,16 @@ NEWS
...
+
+------
+0.11.x
+------
+
+* Drop support for Django
+* Drop support for TurboGears1
+* Drop support for TurboGears2
```
Comment (pypingou) [inline NEWS:15]: + flaks_fas

**Comment by pypingou** [comment]
2 comments but otherwise 👍

File: NEWS (hunk 1 above)
Comment (puiterwijk) [reply NEWS:7]: How about ½.11.0? :D

Comment (pypingou) [reply NEWS:7]: unicode++

**Comment by cverna** [comment]
Closing in favour of #225

**Pull request closed without merging**